Makefile

```diff
@@ -25,25 +39,18 @@ default: docs
 .PHONY: docs
 docs: output/docs.pdf output/docs.html
 
+ifeq "$(strip $(PANDOC))" ''
+output/docs.pdf output/docs.html:
```

Comment:
Can you pickup my second patch in #440 ? I think it would be better to name it as `runtime-spec.pdf/html`, it's what I have to do now before send it to someone else who don't have source code, WDYT?